HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -1577,7 +1578,7 @@ class PerfForesightConsumerType(AgentType):
         MPCmax=1.0,
     )
     time_vary_ = ["LivPrb", "PermGroFac"]
-    time_inv_ = ["CRRA", "Rfree", "DiscFac", "MaxKinks", "BoroCnstArt"]
+    time_inv_ = ["CRRA", "Rfree", "DiscFac", "MaxKinks", "BoroCnstArt", "PerfMITShk" ]
```

Comment:
Let's make `PerfMITShk` something like `tolerance` which the user does not have the option of causing to be a time_varying thing at all.